Computational-Morphogenomics-Group/MarkerMap
# Package MarkerMap for local installation

## Changes
- add MIT license, fix for: https://github.com/Computational-Morphogenomics-Group/MarkerMap/issues/18
- add files to allow creating a package for the code
- update the benchmark_k script to import the package without having to add it to the path
- update the notebooks to import the package without having to add it to the path
- I did not update the notebooks Array_of_Markers and DemoSimpleInterface because they are on colab, so they need to be able to install the package from pip before we can swap over (as far as I am aware)

## Testing
- run through select portions of the notebooks to make sure the imports work correctly
- run through the benchmark script to make sure the imports work there
- future testing: Do a fresh install following the instructions to make sure they work correctly
- ran unit tests

## Doc Changes
- some updates to the readme, specify locally installing the package
